src/main/java/seedu/mamta/Mamta.java

```diff
@@ -0,0 +1,121 @@
+package seedu.mamta;
+import java.io.FileNotFoundException;
+
+
+/**
+ * Main class for the Mamta task manager application.
+ */
+public class Mamta {
+
+    /**
+     * Generates a greeting message.
+     * @return A greeting message.
+     */
+    public static String greet() {
```

Comment:
Perhaps some of this code could be abstracted into other functions from other classes to improve readability?